Objects/Converters/ConverterAutocadCivil/ConverterAutocadCivilShared/ConverterAutocadCivil.Civil.cs

```diff
@@ -1010,12 +1009,19 @@ public Structure StructureToSpeckle(CivilDB.Structure structure)
     };
 
     // assign additional structure props
-    try { speckleStructure["grate"] = structure.Grate; } catch (Exception ex) when (!ex.IsFatal()) { }
```

Comment:
Don't have any idea why it is removed - can you enlight me it is wrapped in somewhere or it is not needed anymore?

Reply:
ah, this is included in the `partData` from my tests